### PR TITLE
Added new script for processing observed V1A and miniseed files repla…

### PR DESCRIPTION
**Pull request by claudio525**
…cing processAllData.py, also some syntax fixes...

### CODE REVIEW

**Review by jpa198 — DISMISSED** [review]
Doesn't seem egregious

File: geoNet/process_observed.py [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    """Retrieves the trace for the specified component"""
+    return [
+        cur_trace.data
+        for cur_trace in traces
+        if cur_trace.stats.channel[-1] == component
+    ][0]
+
+
+def process_v1a_file(ffp: Path, output_format: str, output_type, ix: int, n_files: int):
+    """Processes a single V1A file, to be used with pool.starmap"""
```
Comment (jasonmotha) [inline geoNet/process_observed.py:159]: Processes to what?

File: geoNet/process_observed.py [hunk 2]
```diff
@@ -0,0 +1,370 @@
+"""Processes V1A or miniseed files from observed events and saves these as text files"""
```
Comment (jasonmotha) [inline geoNet/process_observed.py:1]: Probably worth stating that you are leaving the units as is.

**Review by jasonmotha — CHANGES_REQUESTED** [review]

File: geoNet/process_observed.py (hunk 2 above)
Comment (claudio525) [reply geoNet/process_observed.py:1]: Resulting units are always in g?

Comment (claudio525) [reply geoNet/process_observed.py:1]: For V1A the interface returns them as in units g, and for miniseed I divide by g

Comment (jasonmotha) [reply geoNet/process_observed.py:1]: Oh sweet. Are you able to add that to the comment?

**Review by jasonmotha — APPROVED** [review]

File: geoNet/process_observed.py (hunk 2 above)
Comment (claudio525) [reply geoNet/process_observed.py]: Done

**Comment by claudio525** [comment]
@jpa198 From your comment I gather you don't have a problem with this PR? Do you want to approve it, so it can be merged?

**Comment by jasonmotha** [comment]
Oh, one quick thing. Did you want to put your script in a separate directory or move the other stuff elsewhere. So your script is easier to find?

Actually, a short (probably incomplete) README might be the best idea?

**Pull request merged**